pyperf/pyperf_run

```diff
@@ -245,15 +252,18 @@ done
 
 if [ $to_pbench -eq 0 ]; then
 	rm -rf pyperformance
-	git clone https://github.com/python/pyperformance
-	if [ $? -ne 0 ]; then
-		exit_out "Cloning of https://github.com/python/pyperformance failed." 1
+	#
+	# Install pip/pip3
+	#
+	python3 -m ensurepip --upgrade
+	if [[ $? -ne 0 ]]; then
+		exit_out "Failed: python3 -m ensurepip --upgrade" 1
 	fi
-	cd pyperformance
-	git checkout tags/1.0.4
+	python3 -m pip install pyperformance==$PYPERF_VERSION
 	if [ $? -ne 0 ]; then
-		exit_out "Checkout of 1.0.4 failed." 1
+		exit_out "python3 -m pip install pyperformance==$PYPERF_VERSION: failed" 1
 	fi
+	cd pyperformance
```

Comment:
Is this still needed?  Aren't we removing the clone that creates the `pyperformance` dir?